src/main/java/uk/ac/ebi/ena/webin/cli/service/VersionService.java

```diff
@@ -41,8 +49,17 @@ public Version getVersion(String version )
 
     private Version getVersion(String version, boolean test ) {
         RestTemplate restTemplate = new RestTemplate();
-        restTemplate.setErrorHandler(new DefaultErrorHander(WebinCliMessage.VERSION_SERVICE_SYSTEM_ERROR.text()));
```

Comment:
I think we should continue to use setErrorHandler and improve the NotFoundErrorHandler. See previous comment.